### PR TITLE
Follow session - disable ability to change parameter

### PR DESCRIPTION
**Pull request by jankrzempek**
https://trello.com/c/WqkeHh2x/881-follow-session-hide-everything-below-the-session-location-field-default-selections-are-particulate-matter-airbeam

### CODE REVIEW

File: AirCasting/SearchAndFollow/SearchLocation/SearchView.swift [hunk 1]
```diff
@@ -20,32 +20,35 @@ struct SearchView: View {
...
-                    Button {
-                        viewModel.onSensorTap(with: sensor.name)
-                    } label: {
-                        Text(sensor.name)
-                            .padding(5)
-                    }.buttonStyle(MultiSelectButtonStyle(isSelected: sensor.isSelected))
-                        .padding(.bottom, 5)
-                }
-            }
+            #warning("This part was intentionally disabled, refer to https://trello.com/c/WqkeHh2x - to know more")
```
Comment (smialko) [inline AirCasting/SearchAndFollow/SearchLocation/SearchView.swift:23]: Could you please add short explanation? You can quote Michael "openAQ and PurpleAir have revoked free access to their data via their APIs" :) It's a quicker check than going to trello, but you can leave the link, too

File: AirCasting/SearchAndFollow/SearchLocation/SearchViewModel.swift [hunk 2]
```diff
@@ -44,8 +44,8 @@ class SearchViewModel: ObservableObject {
         self.measurementTypes.first(where: { $0.name == param })?.isSelected = true
         switch measurementType(from: param) {
         case .particulateMatter: self.sensorTypes = [
-            .init(isSelected: false, name: SensorType.AB3and2.capitalizedName),
-            .init(isSelected: true, name: SensorType.OpenAQ.capitalizedName),
+            .init(isSelected: true, name: SensorType.AB3and2.capitalizedName),
+            .init(isSelected: false, name: SensorType.OpenAQ.capitalizedName),
             .init(isSelected: false, name: SensorType.PurpleAir.capitalizedName)
```
Comment (smialko) [inline AirCasting/SearchAndFollow/SearchLocation/SearchViewModel.swift:49]: If "openAQ and PurpleAir have revoked free access to their data via their APIs" then should we still have those? 🤔

**Review by smialko — COMMENTED** [review]
Not approving bc I have questions, not requesting changes either - I just need some answers/

File: AirCasting/SearchAndFollow/SearchLocation/SearchViewModel.swift (hunk 2 above)
Comment (jankrzempek) [reply AirCasting/SearchAndFollow/SearchLocation/SearchViewModel.swift:49]: I was thinking about this and came up with some ideas. Disabling the view part, blocks possible interaction with user<>logic. I thought about having it in here for the future (not to reinvent the wheel and be able to easily revert those changes. I am making small improvement in terms of init right now, please tell me what you think

Comment (smialko) [reply AirCasting/SearchAndFollow/SearchLocation/SearchViewModel.swift:47]: Good point, let's leave it then :)

**Review by smialko — APPROVED** [review]
Again - I'm so sorry for keeping you in suspense that long, I'll do better!

**Pull request merged**